core/src/test/scala/org/apache/spark/sql/delta/DeltaWriteConfigsSuite.scala

```diff
@@ -101,7 +101,7 @@ class DeltaWriteConfigsSuite extends QueryTest
   private val dsw_output = new ListBuffer[(String, String, Boolean, Boolean, String)]
   private val dfw_v2_output = new ListBuffer[(String, String, Boolean, Boolean, String)]
   private val dtb_output =
-    new ListBuffer[(String, String, Boolean, Boolean, Boolean, Boolean, String)]
```

Comment:
This test is blowing my head, I think a Tuple6 is maybe better converted into a case class? Also, DeltaWriteConfigSuite takes 3 minutes to run, are you happy for me to split it?

Reply:
@scottsand-db can you confirm you agree on the approach of:
- splitting DeltaWriteConfigSuite in different suites, all within the same Scala files, so that they can be run independently (now the one for the Table Builder runs in 44sec against 180 sec, and it became obvious to me where the problem was). That's the last commit I had and I didn't complete the refactoring before confirming you agree on it
- Replacing Tuple5 and Tuple6 with case classes? I think beyond Tuple3 they should never have made it into the programming language, or at least people should not use them :)

Reply:
@edmondo1984 

> This test is blowing my head, I think a Tuple6 is maybe better converted into a case class? 

Great idea. please do.

> are you happy for me to split it?

This PR is getting big enough. and a test refactoring is a different goal than the specific problem this PR is trying to solve. why don't you split this up in a followup PR?

Reply:
@edmondo1984 - just following up. will you be creating the case class? and updating the column name in the comment-table?

Reply:
I wasn't sure about the scope of the refactoring, so I didn't move forward. Are you ok with this?
- This PR: change case class name and table column
- Other PR: Breaking down the long suite into multiple suites

Reply:
@edmondo1984 - yup that SGTM.